src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -14,6 +14,15 @@
     <Company>MongoDB Inc.</Company>
     <Copyright>Copyright © 2010-present MongoDB Inc.</Copyright>
```

Comment:
If we're using the copyright symbol, perhaps we could also use an endash (–) instead of a hyphen (-)?

Reply:
Comment applies across for all copyright notices